applications/grpc_wallet/src/grpc_interface.rs

```diff
@@ -150,7 +174,18 @@ impl server::WalletRpc for WalletRPC {
         };
 
         let response_body = match self.wallet.text_message_service.get_text_messages_by_pub_key(pub_key) {
-            Ok(msgs) => sort_text_messages(msgs),
+            Ok(mut msgs) => TextMessagesResponseRpc {
+                sent_messages: msgs
+                    .sent_messages
+                    .drain(..)
+                    .map(|m| SentTextMessageRpc::from(m))
```

Comment:
```suggestion
                    .map(|m| m.into())
```

Reply:
This might be simpler, but I'm not sure if it will compile

Reply:
I had some trouble with the map() initially which is why I went the more explicit route but trying it now it does compile. Fixed.